projects/notebooks/testing/prepare_sutest.sh

```diff
@@ -134,7 +142,7 @@ sutest_wait_rhods_launch() {
         ./run_toolbox.py from_config cluster set_project_annotation --prefix sutest --suffix toleration
     fi
 
-    ./run_toolbox.py rhods update_datasciencecluster --enable [dashboard,workbenches]
+    ./run_toolbox.py rhods update_datasciencecluster --enable [dashboard,workbenches,datasciencepipelines]
```

Comment:
can you change this to something like that:
```suggestion
to_enable=dashboard,workbenches
if [[ $(get_config tests.notebooks.test_flavor) == dashboard-scale-test ]]; then
    to_enable="${to_enable},datasciencepipelines"
fi
./run_toolbox.py rhods update_datasciencecluster --enable "[$to_enable]"
```